content/en/docs/concepts/scheduling-eviction/node-pressure-eviction.md

```diff
@@ -105,13 +105,11 @@ does not support other configurations.
 
 Some kubelet garbage collection features are deprecated in favor of eviction:
```

Comment:
So just to be clear,

`--eviction-hard` and `--eviction-soft` are still used to trigger garbage collection.  

We would trigger garbage collection if these signals are passed.  

And we also still use `--image-gc-high-threshold` and `--image-gc-low-threshold` for image garbage collection policies.